packages/api/src/jobs/process-youtube-video.ts

```diff
@@ -199,90 +187,21 @@ export const addTranscriptPlaceholdReadableContent = async (
   return updatedContent.parsedContent?.content
 }
 
-async function readStringFromStorage(
-  bucketName: string,
-  fileName: string
-): Promise<string | undefined> {
-  try {
-    const storage = env.fileUpload?.gcsUploadSAKeyFilePath
-      ? new Storage({ keyFilename: env.fileUpload.gcsUploadSAKeyFilePath })
-      : new Storage()
-
-    const existsResponse = await storage
-      .bucket(bucketName)
-      .file(fileName)
-      .exists()
-    const exists = existsResponse[0]
-
-    if (!exists) {
-      throw new Error(
-        `File '${fileName}' does not exist in bucket '${bucketName}'.`
-      )
-    }
-
-    // Download the file contents as a string
-    const fileContentResponse = await storage
-      .bucket(bucketName)
-      .file(fileName)
-      .download()
-    const fileContent = fileContentResponse[0].toString()
-    return fileContent
-  } catch (error) {
-    // This isn't a catastrophic error it just means the file doesn't exist
-    logger.info('Error downloading file:', error)
-    return undefined
-  }
-}
-
-const writeStringToStorage = async (
-  bucketName: string,
-  fileName: string,
-  content: string
-): Promise<void> => {
-  try {
-    const storage = env.fileUpload?.gcsUploadSAKeyFilePath
-      ? new Storage({ keyFilename: env.fileUpload.gcsUploadSAKeyFilePath })
-      : new Storage()
-
-    const writableStream = storage
-      .bucket(bucketName)
-      .file(fileName)
-      .createWriteStream()
-
-    // Convert the string content to a readable stream
-    const readableStream = new stream.Readable()
-    readableStream.push(content)
-    readableStream.push(null) // Signal the end of the stream
-
-    // Pipe the readable stream to the writable stream to upload the file content
-    await new Promise((resolve, reject) => {
-      readableStream
-        .pipe(writableStream)
-        .on('finish', resolve)
-        .on('error', reject)
-    })
```

Comment:
I think `streaming` is not needed here because we still load the whole content in the memory and push it to the `readStream` at once so it should be the same effect as uploading the whole content at once